### ISSUE TITLE
Question about training loss

### ISSUE THREAD

**Issue by yustaub** [open-mmlab/mmdetection#4514]
It is a nice work of mmdetection! I trained mmdetection on two datasets, one has about 4,000 images and another has 10,000 images, they both trained 12 epoches. The loss of the dataset that contains 4000 images converges to 0.2, and the loss of the dataset that contains 10000 images converges to 0.5 when training finished in epoch 12. I wonder that the different final loss value is caused by dataset, the bigger dataset, will there be bigger loss value? Or I should train more epoches on bigger dataset. Very appreciate for your reply!

**Comment by jshilong** [comment]
If the two data sets are identically distributed, the large data set does require longer training time (but there are also cases where your small data set has been over-fitted)，it would be better to set up a validation dataset to observe convergence.
